### DISCUSSION TITLE
Nanopi R5S怎么都无法启动

### DISCUSSION THREAD

**Discussion by smallprogram** [immortalwrt/immortalwrt#1613] [Q&A]
有没有人用R5S啊，用源码编译，怎么都起不来，从https://downloads.immortalwrt.org/snapshots/targets/rockchip/armv8/ 下载R5S的固件，也一样起不来。Lan口灯不亮，只有Wan口灯亮，获取不到IP，用lean的源码就一点问题没有，有没有那位大佬知道怎么解决吗

**Comment by birdeagles** [comment]
R5S有eMMC吗？我是刷R4SE，刷进eMMC就会出现无法启动的情况，你现在解决了吗？

**Reply by smallprogram** [reply]
解决了，网卡驱动问题

**Reply by birdeagles** [reply]
你是刷进eMMC才有问题的吗？

**Reply by smallprogram** [reply]
不是，无论eMMC还是SD都有问题

**Reply by birdeagles** [reply]
谢谢了，方便具体谈谈你是怎么解决这个驱动问题的吗？

**Reply by smallprogram** [reply]
https://github.com/immortalwrt/immortalwrt/issues/1690

**Reply by birdeagles** [reply]
感谢！
